cypress/integration/search-spec.js

```diff
@@ -48,4 +49,19 @@ describe('Article search', () => {
     searchPage.results().first()
       .children().first().next().should('have.class', 'img')
   })
+
+  it('should show empty result found when search for "adf23cv1"', () => {
+    searchPage.search('adf23cv1')
+    searchPage.getEmptyContent().should('have.text', enJson['no-result-found'])
+  })
+
+  it('center softkey should change when focus on result and search input', () => {
+    searchPage.search('cat')
+    cy.getCenterSoftkeyButton().should('not.have.text')
+    searchPage.results().first()
+    cy.downArrow().downArrow()
```

Comment:
why `downArrow()` twice? any specific case ur looking for ? once works as well

Reply:
no specific reason for this, just some random test